tests/test.program.js

```diff
@@ -202,17 +249,38 @@ describe('program.main', () => {
       });
   });
 
+  it('can get the program version', () => {
+    const fakeVersionGetter = sinon.spy(() => '<version>');
+    const fakeCommands = fake(commands, {
+      build: () => Promise.resolve(),
+    });
+    const projectRoot = '/pretend/project/root';
+    // For some reason, executing --version like this
+    // requires a command. In the real CLI, it does not.
+    return run(['--version', 'build'],
```

Comment:
@kumar303 I looked a bit deeper into this one and I think that the different behavior is related to the `exitProcess` boolean:

when `web-ext` is executed from the command line `exitProcess` is true and `yargs` makes `web-ext` exit just before the commands are checked:

```
...
    return new Promise(
      (resolve) => {
        this.yargs.exitProcess(this._exitProcess);  // <<-- on `web-ext --verbose` yargs exit here
        argv = this.yargs.argv;
        cmd = argv._[0];
        if (cmd === undefined) {
          throw new WebExtError('No sub-command was specified in the args');
        }
...
```